packages/sfui/frameworks/react/hooks/useTrapFocus/useTrapFocus.ts

```diff
@@ -16,6 +16,10 @@ type UseTrapFocusOptions = TabbableOptions &
     initialFocus?: number | `${InitialFocusType}` | false;
     initialFocusContainerFallback?: boolean;
     arrowKeysOn?: boolean;
+    /**
+     * @deprecated Since version 2.3.
+     * @param {boolean} arrowKeysOn - Enabling both `letf` | `up` | `right` | `down` arrow keys.
+     */
```

Comment:
```suggestion
    /**
     * Enabling both `left` | `up` | `right` | `down` arrow keys.
     * @deprecated Since version 2.3. Use arrowKeysLeftRight or/and arrowKeysUpDown options instead.
     */
    arrowKeysOn?: boolean;
```